### PR TITLE
Fix overlapping indexes in Cloister of Frost Battlefields

### PR DESCRIPTION
**Pull request by CatsEyeXI**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Corrects an overlapping index in Cloister of Frost BCNMs that will cause players intending to enter the Trial by Ice BCNM to land in Class Reunion instead.

## Steps to test these changes
!togglegm and Enter each battlefield, see that you will land in the correct one.

### CODE REVIEW

File: scripts/battlefields/Cloister_of_Frost/class_reunion.lua [hunk 1]
```diff
@@ -10,7 +10,7 @@ local content = BattlefieldQuest:new({
     battlefieldId    = xi.battlefield.id.CLASS_REUNION,
     maxPlayers       = 6,
     timeLimit        = utils.minutes(30),
-    index            = 0,
+    index            = 2,
```
Comment (claywar) [inline scripts/battlefields/Cloister_of_Frost/class_reunion.lua]: This should be index 1

Comment (claywar) [reply scripts/battlefields/Cloister_of_Frost/class_reunion.lua]: Ref:
```
    [xi.zone.CLOISTER_OF_FROST] =
    {
        { 0,  480,    0 },   -- Trial by Ice (Quest)
        { 1,  481, 1171 },   -- Class Reunion (Quest)
        { 2,  482, 1545 },   -- Trial-size Trial by Ice (Quest)
    --  { 3,  483,    0 },   -- Waking the Beast (Quest)
        { 4,  484,    0 },   -- Sugar-coated Directive (ASA4)
    --  { 5,    ?,    0 },   -- *Trial by Ice (HTMBF)
    },
```

File: scripts/battlefields/Cloister_of_Frost/trial-size_trial_by_ice.lua [hunk 2]
```diff
@@ -11,7 +11,7 @@ local content = BattlefieldQuest:new({
     maxPlayers       = 1,
     levelCap         = 20,
     timeLimit        = utils.minutes(15),
-    index            = 2,
+    index            = 3,
```
Comment (claywar) [inline scripts/battlefields/Cloister_of_Frost/trial-size_trial_by_ice.lua]: Index 2 is expected here

Comment (CatsEyeXI) [reply scripts/battlefields/Cloister_of_Frost/trial-size_trial_by_ice.lua]: all set thanks

**Review by claywar — APPROVED** [review]

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**